api_facade.py

```diff
@@ -0,0 +1,290 @@
+            "courseId": "swen-344",
+            "numGrades": 40,
+            "numStudents": 40,
+            "instructor": "0001",
+            "students": ["0001", "0002", "0003", "0004"],
+        },
+    'swen-331':
+        {
+            "courseName": "Engineering Secure Software",
+            "courseId": "swen-331",
+            "numGrades": 40,
+            "numStudents": 40,
+            "instructor": "0002",
+            "students": ["0001", "0002", "0003", "0004"],
+        },
+    'swen-440':
+        {
+            "courseName": "Software System Requirements and Architecture",
+            "courseId": "swen-440",
+            "numGrades": 40,
+            "numStudents": 40,
+            "instructor": "0002",
+            "students": ["0001", "0002", "0003", "0004"],
+        },
+    'swen-444':
+        {
+            "courseName": "Human Centered Requirements and Design",
+            "courseId": "swen-444",
+            "numGrades": 40,
+            "numStudents": 40,
+            "instructor": "0001",
+            "students": ["0001", "0002", "0003", "0004"],
+        },
+    'swen-561':
+        {
+            "courseName": "Senior Project I",
+            "courseId": "swen-561",
+            "numGrades": 40,
+            "numStudents": 40,
+            "instructor": "0003",
+            "students": ["0001", "0002", "0003", "0004"],
+        },
+    'swen-562':
+        {
+            "courseName": "Senior Project II",
+            "courseId": "swen-562",
+            "numGrades": 40,
+            "numStudents": 40,
+            "instructor": "0003",
+            "students": ["9999", "9998", "9997", "9996"],
+        },
+}
+
+
+instructors = {
+    "0001":
+        {
+            "name": "Krutz",
+        },
+    "0002":
+        {
+            "name": "Malachowsky",
+        },
+    "0003":
+        {
+            "name": "Vallino",
+        },
+}
+
+
+students = {
+    "9999":
+        {
+            "name": "Lexie",
+        },
+    "9998":
+        {
+            "name": "Sam",
+        },
+    "9997":
+        {
+            "name": "Shayde",
+        },
+    "9996":
+        {
+            "name:" "Yura",
+        }
+}
+
+
+student_courses = {
+    "swen-3449999": {
+        "student_id": "9999",
+        "course_id": "swen-344",
+        "grade": "A",
+        "locked": 0,
+    },
+    "swen-3449998": {
+        "student_id": "9998",
+        "course_id": "swen-344",
+        "grade": "B",
+        "locked": 1,
+    },
+    "swen-3449997": {
+        "student_id": "9997",
+        "course_id": "swen-344",
+        "grade": "C",
+        "locked": 0,
+    },
+    "swen-3449996": {
+        "student_id": "9996",
+        "course_id": "swen-344",
+        "grade": "D",
+        "locked": 1,
+    }
+}
+
+
+comments = {
+    "0001":
+        {
+            "student_course_id": "swen-3449999",
+            "comment": "Thanks",
+            "timestamp": "Yesterday",
+            "author": "Lexie",
```

Comment:
Ah right, and our design doesn't currently support that....What if there was a boolean `isProf` to distinguish the two? I think that would work for our use cases and would be easy to add to the design.